src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -350,22 +358,20 @@ private static void remove(String user, Set<String> projects, Client esClient, S
 		return patterns;
 	}
 	
-	private static void createIndex(String username, String project, Client esClient, String kibanaIndex, boolean use_cdm, String project_prefix, String time_field_name) {
-		
-		DocumentBuilder sourceBuilder = new DocumentBuilder()
-				.title(getIndexPattern(project, project_prefix))
-				.timeFieldName(time_field_name);
+	private void createIndex(String username, String project, Client esClient, String kibanaIndex, String project_prefix, String time_field_name) {
 		
+		final String indexPattern = getIndexPattern(project, project_prefix);
+		String source;
 		if ( project.equalsIgnoreCase(OPERATIONS_PROJECT) || project.equalsIgnoreCase(ADMIN_ALIAS_NAME) )
-			sourceBuilder.operationsFields(use_cdm);
+			source = mappingLoader.getOperationsMappingsTemplate();
 		else if ( project.equalsIgnoreCase(BLANK_PROJECT) )
-			sourceBuilder.blankFields();
+			source = mappingLoader.getEmptyProjectMappingsTemplate();
 		else
-			sourceBuilder.applicationFields(use_cdm);
+		    source = mappingLoader.getApplicationMappingsTemplate();
 		
-		String source = sourceBuilder.build();
```

Comment:
I removed it since the doc builder builds up a document based on various calls to produces the desired outcome.  This change requires the document to be provided complete so there is not a document 'to build'.